.changeset/bright-clouds-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': patch
```

Comment:
```suggestion
'@ithaka/pharos': minor
```

Reply:
@daneah updated in [17f2c7c](https://github.com/ithaka/pharos/pull/416/commits/17f2c7cfeee4ee208e7441921fe91dd501cabd5f).